content/facets.md

```diff
@@ -0,0 +1,138 @@
+
+## Contributing Factors
+
+### 4. Backing Services
+
+By separating backing services across a network connection, it becomes easier to scale application processes across multiple hosts.  Services which are provided only to the local machine require additional setup and possibly replication when compared with network services, and so should be avoided.
+
+### 6. Processes
+
+Running the application as one or more stateless processes makes it easier to increase and decrease the number of instances of the application, without requiring coordination between the application and the platform layer to choose an instance, and without requiring coordination between application servers to handle loss of state.
+
+### 7. Port Binding
+
+Exposing services via port binding (or a higher-level service, such as HTTP) allows the platform to understand and control which instances are *active*, and which may be scaled down or are available to receive a new request when scaling up.
+
+### 8. Concurrency
+
+Scale-out via concurrency is the cornerstone of scalable applications. By ensuring that each instance is an independent share-nothing process which is designed to run multiple instances, application authors can enable scalability without needing to invest deeply in the complexity of distributed applications.
+
+### 9. Disposability
+
+Automatic, well-managed startup and shutdown is essential for the automation of scaling via the concurrency model.  While an application could be built to scale via the concurrency model without assuming instances are easily replaced, it would be difficult to connect such an application to infrastructure which could scale it automatically.
+
+# Observable
+
+## What
+
+Observable applications make it possible to understand and measure the state of the running application by making measurements without disturbing the running code.  Examples of these measurements can include aggregate metrics, logged application messages, traces and profiles captured during application execution, and more.
+
+## Why
+
+While local development instances may be easy to debug and interact with by pausing and interacting with using a debugger, scalable distributed systems in production often need to be able to operate continuously, even while debugging issues.  Applications which provide a suite of interfaces to understand the internal application state while operating are easier to fix and maintain than applications which need interactive debugging, particularly for issues caused by data differences between environments.
+
+## Contributing Factors
+
+### 4. Backing Services?
```

Comment:
I'm not 100% on this one. I feel like if we comment on this it should be included in the updated observability factor

Reply:
Live discussion we discussed adding a little more detail about what is meant here, along with an example. 👍 to that idea